stream-chat-android-client/src/main/java/io/getstream/chat/android/client/header/VersionPrefixHeader.kt

```diff
@@ -0,0 +1,10 @@
+package io.getstream.chat.android.client.header
+
+import io.getstream.chat.android.core.internal.InternalStreamChatApi
+
+@InternalStreamChatApi
+public enum class VersionPrefixHeader(public val prefix: String) {
+    CORE_ANDROID("stream-chat-android-"),
+    COMPOSE("stream-chat-compose-")
+    // TODO - @Rafal - we'll need to add a value for the old UI component too
```

Comment:
Could you add the one for `UI-Components` module?
The old UI should't need it, we will remove it soon.

I think you should be able to init it on `ChatUIInitializer` [class](https://github.com/GetStream/stream-chat-android/blob/main/stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/common/internal/ChatUIInitializer.kt)